src/server.js

```diff
@@ -280,7 +283,72 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_ADDRESS}/geocode`,
+  async (request, response) => {
+    const address = encodeAddress(request.params[URL_PARAM_ADDRESS]);
+
+    const geocodeRequest =
+      "https://maps.googleapis.com/maps/api/geocode/json?address=" +
+      address +
+      "&key=" +
+      env.API_KEY_GEOCODE;
+
+    try {
+      const geocodeResponse = await (await fetch(geocodeRequest)).json();
+      const geocodeResponseStatus = geocodeResponse.status;
+
+      if (geocodeResponseStatus !== "OK") {
+        console.error(
+          "Geocoding error occured. Api response status: " +
+            geocodeResponseStatus
+        );
+        response
+          .status(500)
+          .json({ status: 500, error: { type: geocodeResponseStatus } });
+      } else {
+        response.json({
+          status: 200,
+          data: geocodeResponse.results[0].geometry.location,
+        });
+      }
+    } catch (err) {
+      console.error(err);
+      response
+        .status(500)
+        .json({ status: 500, error: { type: ERROR_GEOCODING_FAILED } });
+    }
+  }
+);
+
+function encodeAddress(address) {
+  let formattedAddress = encodeURIComponent(address);
```

Comment:
Nit: The `replace` calls can probably be chained as follows: `a.replace("!", "%21").replace(...).replace(...)`

Reply:
Done.